frontend/packages/dev-console/src/utils/__tests__/pipeline-actions.spec.ts

```diff
@@ -36,14 +36,22 @@ export const actionPipelineRuns: PipelineRun[] = [
     kind: 'PipelineRun',
     metadata: { name: 'winterfell', namespace: 'corazon' },
     spec: { pipelineRef: { name: 'sansa-stark' } },
-    status: { creationTimestamp: '31', conditions: [{ type: 'Succeeded', status: 'True' }] },
+    status: {
+      pipelineSpec: { tasks: [] },
+      creationTimestamp: '31',
+      conditions: [{ type: 'Succeeded', status: 'True' }],
+    },
   },
   {
     apiVersion: 'abhiapi/v1',
     kind: 'Pipeline',
     metadata: { name: 'dragonstone', namespace: 'corazon' },
     spec: { pipelineRef: { name: 'danaerys-targaeryen' } },
-    status: { creationTimestamp: '31', conditions: [{ type: 'Succeeded', status: 'Unknown' }] },
+    status: {
+      pipelineSpec: { tasks: [] },
+      creationTimestamp: '31',
+      conditions: [{ type: 'Succeeded', status: 'Unknown' }],
+    },
```

Comment:
Well this is duplicate data from `frontend/packages/dev-console/src/components/pipelines/modals/common/__tests__/utils.spec.ts` 😞 

Think we could move this data into `pipeline-data.ts` and then import it back?

Reply:
Removed this mock data and used the data from `pipeline-data.ts`  and also refactored `utils.spec.ts` to avoid duplication of mock data.